classes/security/Validation.inc.php

```diff
@@ -430,6 +430,32 @@ static function canAdminister($administeredUserId, $administratorUserId) {
 		// There were no conflicting roles. Permit administration.
 		return true;
 	}
+
+	/**
+	 * Validation process for imported passwords
+	 * @param $userToImport User ByRef. The user that is being imported. 
+	 * @param $encryption string null, sha1, md5 (or any other encryption algorithm defined)
+	 * @return string if a new password is generated, the function returns it.
+	 */
+	function importUserPasswordValidation(&$userToImport, $encryption) {
```

Comment:
The reference to `$userToImport` isn't needed.

Reply:
Done.